docs/user/compare_flights.rst

```diff
@@ -0,0 +1,226 @@
+Compare Flights
+===============
+
+This example demonstrates how to use the rocketpy ``CompareFlights`` class.
+This class has many applications, including the comparison of different flight
+setups for a single rocket, the simulation of deployable systems, and the
+multi-stage rocket analysis.
+
+Importing classes
+-----------------
+
+We will start by importing the necessary classes and modules:
+
+.. jupyter-execute::
+
+      from rocketpy.plots.compare import CompareFlights
+      from rocketpy import Environment, Flight, Rocket, SolidMotor
+      from datetime import datetime, timedelta
+
+
+Create Environment, Motor and Rocket
```

Comment:
A `see also` to the first simulation page would be useful here

Reply:
done [ca74af0](https://github.com/RocketPy-Team/RocketPy/pull/576/commits/ca74af040125039dcada325d740892fc83cd21a4)